homeassistant/components/amberelectric/sensor.py

```diff
@@ -104,6 +104,29 @@ def extra_state_attributes(self) -> dict[str, Any] | None:
             data["range_min"] = format_cents_to_dollars(interval.range.min)
             data["range_max"] = format_cents_to_dollars(interval.range.max)
 
+        if interval.advanced_price is not None:
+            if interval.channel_type == ChannelType.FEEDIN:
```

Comment:
[Non blocking]: Since both halves of the if are so similar, you could create a multiplier variable then use the rest of the logic only once.